contracts/GnosisWallet.sol

```diff
@@ -36,59 +36,50 @@ contract MultiSigWallet {
     }
 
     modifier onlyWallet() {
-        if (msg.sender != address(this))
-            throw;
+        require(msg.sender == address(this));
```

Comment:
For Gnosis, let's just use fresh upstream file

Reply:
Will use upstream file once upstream has merged my pull request gnosis/MultiSigWallet#12 that eliminates throw and other warnings, thanks.